Public/components/cards/cards.css

```diff
@@ -0,0 +1,92 @@
+@import url("/Public/components/alert/alert.css");
+.basic-card{
+    width:fit-content;
+    border-radius: 8px;
+    background-color: white;
+    box-shadow: 0px 4px 8px black;
+    
+}
+.basic-card:hover{
+    box-shadow: 0px 4px 18px black;
+}
+.card-image{
+    display: flex;
+}  
+.card-image-button .btn-heart{
+    background-color: white;
+    position: relative;
+    top: -245px;
+    right: -195px;
+}
+.card-image-button .btn-heart:hover{
+    background-color: var(--primary-color);
+}
+.card-text-container{
+    display: flex;
+    flex-direction: column;
+    margin-top: -50px;
```

Comment:
try to use rems instead of px, you can use online px to rem converter if you need any specific unit.